packages/core-components/report-testUtils.api.md

```diff
@@ -3,8 +3,31 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
+// @public (undocumented)
+export type Breakpoint = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+
+// Warning: (tsdoc-undefined-tag) The TSDoc tag "@xampple" is not defined in this configuration
+// Warning: (tsdoc-undefined-tag) The TSDoc tag "@xample" is not defined in this configuration
+//
 // @public
-export function mockBreakpoint(options: { matches: boolean }): void;
+export function mockBreakpoint(
+  options:
+    | {
+        matches: boolean;
+      }
+    | {
+        initialBreakpoint?: Breakpoint;
+        queryBreakpointMap?: Record<string, Breakpoint>;
+      },
```

Comment:
Might be worth experimenting with using an overload instead of the union. That lets use have separate documentation for the two for example. Whichever one works the best when looking at the function in the editor and exploring call options

Reply:
Done 🙌🏻